dashboard/test/test_helper.rb

```diff
@@ -333,3 +333,7 @@ def write(json)
     @records << json
   end
 end
+
+def json_response
+  ActiveSupport::JSON.decode @response.body
```

Comment:
any particular reason to use `ActiveSupport::JSON.decode` instead of just `JSON.parse`?